splinterd/man/splinterd.1.md

```diff
@@ -149,6 +147,9 @@ OPTIONS
 : Specifies the endpoint for daemon-to-daemon communication between Splinter
   nodes, using the format `tcp://ip:port`. (Default: 127.0.0.1:8044.)
```

Comment:
`tcp://` should be changed to `protocol_prefix://`. we should also explain how tcps:// creates a TLS connection and tcp:// connection a TCP connection. This is important because if you want to accept both types you need to provide an endpoint for each (to create a listener for)

Reply:
I'd like to make this change in a separate PR.  Is that okay?

Reply:
yes